frontend/pages/news/article/[id]/index.tsx

```diff
@@ -32,6 +33,11 @@ export default function ArticlePage() {
 
   return (
     <NoTitleLayout>
```

Comment:
I think view might be a bit confusing. The alert is neat and shows that it's a draft, but there's not really any relevant action, i,e navigating to the request menu, buttons for approving/rejecting, etc. Maybe there's no point in linking to /news/article/[id] from /news/requests?

![image](https://github.com/Dsek-LTH/member-page/assets/26229521/5ee62951-7bf9-41f8-a2ea-e1dcc4f88226)